### ISSUE TITLE
FNX-499 ⁃ When launching Fenix, Enable private browsing button is focused instead of Search or address

### ISSUE THREAD

**Issue by asadotzler** [mozilla-mobile/fenix#2142]
## Steps to reproduce
Launch Fenix

### Expected behavior
Search or enter address is focused

### Actual behavior
Enable private browsing button is focused

### Device information
* Android device: ? Galaxy S6
* Fenix version: ? Firefox Preview 1.0.1918 (Build #11181249 68.0a1-20190422094240)

**Comment by asadotzler** [comment]
Would love to get this but we'd probably ship without it.

**Comment by fobo66** [comment]
I'd like to work on it. Found a root cause, preparing a fix now

**Comment by chiragmantri** [comment]
@asadotzler is this issue fixed? or is there any work item left for this?

**Comment by fobo66** [comment]
I've fixed this, it appears to be working fine now

**Comment by sv-ohorvath** [comment]
Partially fixed. I logged a follow-up bug since this one has already landed in RC 1.2: https://github.com/mozilla-mobile/fenix/issues/4447.

**Comment by mcarare** [comment]
The fix also causes  #4489, I think it should be revisited. Also, IMO the focus change is against good practices in accessibility.

**Comment by ekager** [comment]
Okay @mcarare I can back out the change and maybe we can make a meta ticket about home screen a11y issues as we look at redesign options

**Comment by ekager** [comment]
I'm going to undo these changes because it seems to cause other issues (#4447 and #4489), so let's reopen this and we can bring it up as an accessibility concern about the homescreen.

**Comment by fobo66** [comment]
Looks like this issue cannot be properly fixed (without accessibility focus change) because of homescreen layout hierarchy

**Comment by lime124** [comment]
oops forgot to remove needs ux label - i've added it to an a11y audit so we can look at this then.

**Comment by mcarare** [comment]
This is fixed, as a result of the changes implemented as per comment: https://github.com/mozilla-mobile/fenix/issues/1718#issuecomment-566674558.
